orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -118,7 +120,11 @@ func getMagmadMconfig(
 	if err != nil {
 		return nil, err
 	}
-
+	orc8rVersion, res := os.LookupEnv("VERSION_TAG")
```

Comment:
At a high level, we want our components (here, servicers), to be as stateless as possible -- where everything that breaks the statelessness is passed in as a dependency during component creation/initialization

That to say: ideally we would only have to read this env var once (during servicer creation), rather than once per request

Reply:
ideally if we want our services to be stateless, then we should store everything breaking statelessness in a distributed store(redis?) Because if i were to run another instance of orc8r in a different environment, that environment may or may not have this env variable set, so passing it during initialization doesnt really help with statelessness.